docs/src/components/main-nav.tsx

```diff
@@ -17,7 +17,7 @@ export interface NavItem {
 }
 
 export function MainNav(props: { items: NavItem[] }) {
-  const pathname = usePathname();
+  const segment = useSelectedLayoutSegment() ?? "/";
```

Comment:
i don't think this is what we want - this will return the last segment in the path: https://nextjs.org/docs/app/api-reference/functions/use-selected-layout-segment#returns

Reply:
nvm this does look right, just think the fallback state is wrong 

<img width="643" alt="CleanShot 2023-07-17 at 09 01 00@2x" src="https://github.com/t3-oss/t3-env/assets/51714798/02eec1e2-82cf-4be3-9b9b-cc5e2b202194">

Reply:
sorry I don't understand the issue